test/Sentry.NLog.Tests/SentryTargetTests.cs

```diff
@@ -769,15 +779,15 @@ public void GetUserFromLayouts_PropertiesMapped()
             var sentryTarget = factory.Configuration.FindTargetByName<SentryTarget>("sentry");
             sentryTarget.User = new SentryNLogUser
             {
-                Username = "${logger:shortName=true}"
+                Username = "${logger:shortName=true}",
             };
+            sentryTarget.User.Other.Add(new TargetPropertyWithContext("Gender", "${event-properties:Gender}") { IncludeEmptyValue = false });
```

Comment:
Do we have a test that verifies a value does end up on the Sentry User?

Reply:
Nope. Just wanted to code-coverage to be complete.

Reply:
Now changed the test, so it actually test output value is generated.